packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -150,7 +150,7 @@ describeNoCompat("GC DataStore Tombstoned When It Is Sweep Ready", (getTestObjec
         // Modifying a testDataObject substantiated from the request pattern should fail!
         assert.throws(() => testDataObject2._root.set("send", "op"),
             (error) => {
-                const correctErrorType = error.errorType = "dataCorruptionError";
```

Comment:
Ha! Surprised there's no lint rule about this.

Reply:
Yep, there should have been